addons/mail/models/discuss/discuss_channel.py

```diff
@@ -720,7 +720,18 @@ def _channel_info(self):
             return []
         channel_infos = []
         # sudo: discuss.channel.rtc.session - reading sessions of accessible channel is acceptable
-        rtc_sessions_by_channel = self.sudo().rtc_session_ids._mail_rtc_session_format_by_channel()
+        rtc_sessions_by_channel = self.sudo().rtc_session_ids._mail_rtc_session_format_by_channel({
+            "id": True,
+            "channelMember": {
+                "id": True,
+                "channel": {},
+                "persona": {"partner": {"id", "name", "im_status"}, "guest": {"id", "name", "im_status"}},
+            },
+            "isCameraOn": True,
+            "isDeaf": True,
+            "isSelfMuted": True,
+            "isScreenSharingOn": True,
```

Comment:
Can't we find a better way to not have to repeat all of this?
Maybe a specific method?
Maybe a better default param possiblity?

Reply:
I changed the formatter so that `id` and `channelMember` are always there